boards/adi/adi_eval_adin1110ebz/Kconfig.adi_eval_adin1110ebz

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 BayLibre
```

Comment:
revert image change, the image size error is to be ignored, reducing an image size actually increases the repo size

Reply:
Done